.github/workflows/ci.yml

```
@@ -101,7 +101,7 @@ jobs:
           verbose: false
           token: ${{ secrets.CODECOV_TOKEN }}
       
-      # Test probably delete this
+      # Smoke test. Shows end to end run with out crashing.
       - name: Smoke Test
         shell: bash -l {0}
-        run: python run_stac.py
\ No newline at end of file
+        run: python run_stac.py stac=stac_synth_data model=synth_data
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)